### DISCUSSION TITLE
Когда включаю Lantern, ноут не подключается к интенету

### DISCUSSION THREAD

**Discussion by 87iecfl** [getlantern/lantern-forum-ru#11] [Q&A]
Скачал последнюю версию и купил лицензию Pro, поработало один раз, на второй раз уже нет.  Что делать и возможно ли сделать возврат средств если не будет решения ?

**Comment by rtx2014** [comment]
Когда теряет соединение, обычно, перегрузка ОС помогает, во всяком случае, мне.
Если не работает и после перегрузки OC, то ставлю версию 7.4.0, которая почти сразу же обновляется до 7.6.2, но до перезапуска работает стабильно.
